src/tools/mesh_tester.cpp

```diff
@@ -250,6 +269,124 @@ std::vector< std::pair<int, int> > naiveIntersectionAlgorithm(
   return retval;
 }
 
+template < typename ExecSpace >
+std::vector< std::pair<int, int> > naiveIntersectionAlgorithm(
+  mint::Mesh* surface_mesh,
+  std::vector<int> & degenerate)
+{
+  SLIC_INFO("Execution Space is " <<
+    axom::execution_space< ExecSpace >::name());
+
+  // Get allocator
+  int allocatorID = axom::execution_space< ExecSpace >::allocatorID();
+  umpire::Allocator allocator = axom::getAllocator(allocatorID);
```

Comment:
Umpire usage should be guarded by ``AXOM_USE_UMPIRE`` since you are not strictly requiring umpire as a precondition for the ``mesh_tester`` app.

@gzagaris -- is there a clean way to set this up with umpire allocators when umpire is available and using the default allocation when it is not?